packages/serialize-query-params/src/updateLocation.ts

```diff
@@ -10,6 +10,24 @@ const JSON_SAFE_CHARS = `{}[],":`
   .split('')
   .map((d) => [d, encodeURIComponent(d)]);
 
```

Comment:
I pulled this out into a named function to allow focusing in on the task at hand